src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -871,6 +873,14 @@ export function findMeasureGroupInDimensions(dimensions: Execution.IResultDimens
         });
 }
 
+export function findMeasureGroup(
```

Comment:
This function seems either superfluous or not so well named to me. I'd suggest 
a) use `findMeasureGroupInDimensions(executionResponse.dimensions)`
b) name it better, for instance `findMeasureGroupInResponse`
I prefer `a)`

Reply:
Did a)